src/main/java/friday/storage/Storage.java

```diff
@@ -0,0 +1,86 @@
+package friday.storage;
+
+import friday.task.Deadline;
+import friday.task.Event;
+import friday.task.Task;
+import friday.task.Todo;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private final Path savePath;
+
+    public Storage(Path path) {
+        this.savePath = path;
+    }
+
+    public List<Task> load() {
+        List<Task> list = new ArrayList<>();
+        try {
+            if (!Files.exists(savePath)) {
+                if (savePath.getParent() != null) Files.createDirectories(savePath.getParent());
+                return list;
+            }
+            try (BufferedReader br = Files.newBufferedReader(savePath, StandardCharsets.UTF_8)) {
```

Comment:
Nit: Would it be more readable if this was not abbreviated?

```suggestion
            try (BufferedReader reader = Files.newBufferedReader(savePath, StandardCharsets.UTF_8)) {
```